package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hmrc-frontend",
-  "version": "5.4.0",
+  "version": "5.7.0",
```

Comment:
@kyle-bowden needs an `npm install` so package-lock.json gets updated too - but just to check, it's showing a jump of three versions - shouldn't it say 5.5.0? Latest released version is 5.5.0 - though I can see that changelog has 5.6.0 - not sure what's happened here with versions, might want to investigate this before we merge

Reply:
I guess this is the "docs changes don't bump versions" stuff maybe, even though we define the version explicitly I guess it's still short circuiting